deploy/k3d.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: k3d.io/v1alpha2
+kind: Simple
+servers: 1
+network: k3d
+kubeAPI:
+  hostIP: "0.0.0.0"
+  hostPort: "6443"
+volumes:
+  - volume: /prometheus:/prometheus
+    nodeFilters:
+      - server[0]
+  - volume: /grafana:/grafana
+    nodeFilters:
+      - server[0]
+ports:
+  - port: 30000:30000
+    nodeFilters:
+      - server[0]
+  - port: 30080:30080
+    nodeFilters:
+      - server[0]
+  - port: 30088:30088
+    nodeFilters:
+      - server[0]
+  - port: 32000:32000
+    nodeFilters:
+      - server[0]
+  - port: 32080:32080
+    nodeFilters:
+      - server[0]
+  - port: 32088:32880
+    nodeFilters:
+      - server[0]
+
+options:
+  k3d:
+    wait: true
+    timeout: "60s"
+  k3s:
+    extraServerArgs:
+      - --tls-san=127.0.0.1
+    extraAgentArgs: []
+  kubeconfig:
+    updateDefaultKubeconfig: true
+    switchCurrentContext: true
```

Comment:
linter - missing EOL

you can set your editor / git to do this automatically

sudo git config --system core.whitespace blank-at-eol,blank-at-eof,space-before-tab